shared/gh/js/views/gh.listview.js

```diff
@@ -181,6 +220,12 @@ define(['gh.utils', 'gh.api.orgunit', 'gh.constants'], function(utils, orgunitAP
     var addBinding = function() {
         // Toggle a list item
         $('body').on('click', '.gh-toggle-list', toggleList);
+
+        // Hide the popover window
+        $('body').on('mouseout', '.list-group-item .fa-link', dismissSeriesPopover);
+        // Show extra information for the borrowed series
+        $('body').on('mouseover', '.list-group-item .fa-link', setUpSeriesPopover);
```

Comment:
I think we should make the containing span element the trigger as it has a hover state but doesn't necessarily show the popover yet then.